project/src/main/kotlin/org/ionproject/core/common/LogMessages.kt

```diff
@@ -39,25 +44,24 @@ object LogMessages {
     const val unknownError = "It seems the error you received isn't yet documented, inform the developers."
 
     //Authentication Messages
-    fun forAuthError(authenticationMode: String, reason: String) =
-        "[ID:${logMessageId++}] [AUTHENTICATION] | TYPE:[$authenticationMode] | RESULT:[ACCESS DENIED] | REASON:[$reason]"
+    fun forAuthError(reason: String) =
+        "$unsuccessAuthEvent [rid:${MDC.get(REQUEST_ID)} | uid:${logMessageId.getAndIncrement()}] | REASON:[$reason]"
 
-    fun forAuthErrorDetail(authenticationMode: String, tokenHash: String, method: String, requestUrl: String, reason: String) =
-        "[ID:${logMessageId++}] [AUTHENTICATION] | TYPE:[$authenticationMode] | TOKEN_HASH:[${tokenHash}] " +
-        "| METHOD:[${method}] | LOCATION:[${requestUrl}] | RESULT:[ACCESS DENIED] " +
+    fun forAuthErrorDetail(tokenHash: String, reason: String) =
+        "$unsuccessAuthEvent [rid:${MDC.get(REQUEST_ID)} | uid:${logMessageId.getAndIncrement()}] | TOKEN_HASH:[${tokenHash}] " +
         "| REASON:[$reason]"
 
-    fun forAuthSuccess(authenticationMode: String, tokenHash: String, method: String, url: String) =
-        "[ID:${logMessageId++}] [AUTHENTICATION] | TYPE:[$authenticationMode] | TOKEN:[${tokenHash}] | METHOD:[${method}] | LOCATION:[${url}] | RESULT:[ACCESS GRANTED]"
+    fun forAuthSuccess(tokenHash: String) =
+        "$successAuthEvent [rid:${MDC.get(REQUEST_ID)} | uid:${logMessageId.getAndIncrement()}] | TOKEN:[${tokenHash}]"
 
     //Exception Messages
     fun forException(url: String, detail: String) =
-        "[ID:${logMessageId++}] [EXCEPTION] | LOCATION:[$url] | DETAIL:[$detail]"
+        "$exceptionEvent [rid:${MDC.get(REQUEST_ID)} | uid:${logMessageId.getAndIncrement()}] | LOCATION:[$url] | DETAIL:[$detail]"
 
     //Logger interceptor message
-    fun forLoggerAccessMessage(ip: String, method: String, url: String, startTime: Long) =
-        "[ID:${logMessageId++}] |  Request-Id:${MDC.get(REQUEST_ID)} |[INFORMATION] | IP:[${ip}] | Method:[${method}] | Endpoint:[${url}] | Timestamp:[${startTime}]"
+    fun forLoggerAccessMessage(ip: String, method: String, url: String) =
+        "$httpStartEvent [rid:${MDC.get(REQUEST_ID)} | uid:${logMessageId.getAndIncrement()}] | IP:[${ip}] | Method:[${method}] | Endpoint:[${url}]"
```

Comment:
Don't use Endpoint and the url. Follow what is told here https://github.com/i-on-project/core/pull/222#discussion_r452330550.